src/core/public/chrome/chrome_service.mock.ts

```diff
@@ -36,6 +36,7 @@ import { getLogosMock } from '../../common/mocks';
 const createSetupContractMock = () => {
   return {
     registerCollapsibleNavHeader: jest.fn(),
+    getUseCaseEnabled: jest.fn(),
```

Comment:
Perhaps we need to call it `getNavGroupEnabled`? navGroup seems to be a term more acceptable by others from PR #7060

Reply:
Sure, will refactor all use case term to group in the next commit.